runtime/mainnet/src/config/xcm.rs

```diff
@@ -214,56 +244,746 @@ impl cumulus_pallet_xcm::Config for Runtime {
+
+		#[test]
+		fn does_not_have_aliasers() {
+			assert_eq!(
+				TypeId::of::<<XcmConfig as xcm_executor::Config>::Aliasers>(),
+				TypeId::of::<Nothing>(),
+			);
+		}
+
+		#[test]
+		fn asset_claims_via_xcm() {
+			assert_eq!(
+				TypeId::of::<<XcmConfig as xcm_executor::Config>::AssetClaims>(),
+				TypeId::of::<PolkadotXcm>(),
+			);
+		}
+
+		#[test]
+		fn asset_exchanger_is_disabled() {
+			assert_eq!(
+				TypeId::of::<<XcmConfig as xcm_executor::Config>::AssetExchanger>(),
+				TypeId::of::<()>(),
+			);
+		}
+
+		#[test]
+		fn asset_locker_is_disabled() {
+			assert_eq!(
+				TypeId::of::<<XcmConfig as xcm_executor::Config>::AssetLocker>(),
+				TypeId::of::<()>(),
+			);
+		}
+
+		#[test]
+		fn uses_local_asset_transactor() {
+			assert_eq!(
+				TypeId::of::<<XcmConfig as xcm_executor::Config>::AssetTransactor>(),
+				TypeId::of::<LocalAssetTransactor>(),
+			);
+		}
+
+		#[test]
+		fn traps_assets_via_xcm() {
+			assert_eq!(
+				TypeId::of::<<XcmConfig as xcm_executor::Config>::AssetTrap>(),
+				TypeId::of::<PolkadotXcm>(),
+			);
+		}
+
+		#[test]
+		fn configures_barrier() {
+			assert_eq!(
+				TypeId::of::<<XcmConfig as xcm_executor::Config>::Barrier>(),
+				TypeId::of::<Barrier>(),
+			);
+		}
+
+		#[test]
+		fn call_dispatcher_is_runtime_call() {
+			assert_eq!(
+				TypeId::of::<<XcmConfig as xcm_executor::Config>::CallDispatcher>(),
+				TypeId::of::<RuntimeCall>(),
+			);
+		}
+
+		#[test]
+		fn fee_manager_resolves_to_treasury() {
+			assert_eq!(
+				TypeId::of::<<XcmConfig as xcm_executor::Config>::FeeManager>(),
+				TypeId::of::<
+					XcmFeeManagerFromComponents<
+						WaivedLocations,
+						SendXcmFeeToAccount<
+							<XcmConfig as xcm_executor::Config>::AssetTransactor,
+							TreasuryAccount,
+						>,
+					>,
+				>(),
+			);
+		}
+
+		#[test]
+		fn hrmp_accepted_handler_is_disabled() {
+			assert_eq!(
+				TypeId::of::<<XcmConfig as xcm_executor::Config>::HrmpChannelAcceptedHandler>(),
+				TypeId::of::<()>(),
+			);
+		}
+
+		#[test]
+		fn hrmp_closed_handler_is_disabled() {
+			assert_eq!(
+				TypeId::of::<<XcmConfig as xcm_executor::Config>::HrmpChannelClosingHandler>(),
+				TypeId::of::<()>(),
+			);
+		}
+
+		#[test]
+		fn hrmp_new_request_handler_is_disabled() {
+			assert_eq!(
+				TypeId::of::<<XcmConfig as xcm_executor::Config>::HrmpNewChannelOpenRequestHandler>(
+				),
+				TypeId::of::<()>(),
+			);
+		}
+
+		#[test]
+		fn trusted_reserves_are_provided() {
+			assert_eq!(
+				TypeId::of::<<XcmConfig as xcm_executor::Config>::IsReserve>(),
+				TypeId::of::<TrustedReserves>(),
+			);
+		}
+
+		#[test]
+		fn does_not_configure_teleporters() {
+			assert_eq!(
+				TypeId::of::<<XcmConfig as xcm_executor::Config>::IsTeleporter>(),
+				TypeId::of::<()>(),
+			);
+		}
+
+		#[test]
+		fn limits_assets_in_holdings() {
+			assert_eq!(
+				<<XcmConfig as xcm_executor::Config>::MaxAssetsIntoHolding as Get<u32>>::get(),
+				64,
+			);
+		}
+
+		#[test]
+		fn message_exporter_is_dissabled() {
```

Comment:
typo in disabled.

Reply:
thanks!

Reply:
[2c256cb](https://github.com/r0gue-io/pop-node/pull/455/commits/2c256cbb26f474991261da2bd306c7e21cc06c0e)